.github/workflows/maven-test.yml

```diff
@@ -0,0 +1,31 @@
+name: Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Check out Git repository
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      # Set up environment with Java and Maven
+      - name: Setup JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      # Set up dependency cache
+      - name: Cache local Maven repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+
+      # Build & Test
+      - name: Build & Test with Maven
+        run: mvn -B -U clean install #-s ./.github/workflows/settings.xml
```

Comment:
why is the settings.xml in comment? if on purpose add a comment

Reply:
DONE